db/compaction_picker.cc

```diff
@@ -142,6 +110,39 @@ void CompactionPicker::ReleaseCompactionFiles(Compaction* c, Status status) {
   }
 }
 
+bool CompactionPicker::FindIntraL0Compaction(
```

Comment:
Is `FindIntraL0Compaction` the same as before except that it becomes a member function? Does it need access to any of the members of `CompactionPicker`? If not, maybe it does not have to be a member function. Let me know if you have other thoughts.

Reply:
Thanks for reviewing, @riversand963. 

`FindIntraL0Compaction` is the same as before, and does not need access to any fields of `CompactionPicker`. I guess you have a good point that I need not make it a member function. I only made it a member of CompactionPicker class as `FIFOCompactionPicker` also makes use of it, in addition to Level. 
Let me try moving it back out of `CompactionPicker`.